src/app/participant-list/participant-list.component.ts

```diff
@@ -533,6 +538,21 @@ export class ParticipantListComponent implements OnInit {
     );
   }
 
+  private addDynamicFieldDefaultColumns(defaultColumn: any) {
+    outer: for (let sourceColumnGroup of Object.keys(this.sourceColumns)) {
+      for (let currentFilter of this.sourceColumns[sourceColumnGroup]) {
```

Comment:
In outer loop you could use `Object.values(this.sourceColumns)` and you would have values directly instead of iterating though the keys and getting values by keys.

Reply:
I tried it, but it wouldn't understand the values to be Arrays, so I decided to stick with this approach

Reply:
```
outer: for (let sourceColumnGroup of Object.values(this.sourceColumns)) {
      for (let currentFilter of sourceColumnGroup as Array<Filter>) {
```

this casting should work